pkgs/tools/misc/lesspipe/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, perl }:
+
+stdenv.mkDerivation rec {
+  name = "lesspipe";
+
+  version = "1.82";
+
+  propagatedBuildInputs = [ perl ];
```

Comment:
What is this for? Build inputs primarily affect the environment in which the build process runs; a propagated build input X of A means that if B adds A to its build environment, X is added as well. If you mean to indicate that Perl is a required run-time dependency, it is probably better to patch the path to the perl interpreter into the script itself (this will ensure that Perl is included in lesspipe's runtime closure).